API.Tests/Services/SeriesServiceTests.cs

```diff
@@ -1119,4 +1137,176 @@ public void GetFirstChapterForMetadata_NonBook_ShouldReturnVolume1_WhenFirstChap
     }
 
     #endregion
+
+    #region SeriesRelation
+    [Fact]
+    public async Task UpdateRelatedSeries_ShouldAddAllRelations_And_DeleteAllRelations()
```

Comment:
I would copy/paste the code and do the Delete in a separate test to keep it clean.

Reply:
@ThePromidius still waiting on the comments to be taken care of

Reply:
I completely missed this. Will do